remix-solidity/src/compiler/compiler.js

```diff
@@ -177,6 +177,19 @@ function Compiler (handleImportCall) {
     return null
   }
 
+  this.ldCompilationResult = (file, source, languageVersion, data) => {
```

Comment:
is this still needed?

Reply:
@yann300 Oh were you referring to the log statements? removing those now